XcodeServerSDK/Utils/HTTPUtils.swift

```diff
@@ -53,13 +53,13 @@ public class HTTP {
                         
                     case let s where s.rangeOfString("application/json") != nil:
                         
-                        let (json: AnyObject!, error) = JSON.parse(data)
-                        let headers = httpResponse.allHeaderFields
+                        let (json, error) = JSON.parse(data!)
+                        // let headers = httpResponse.allHeaderFields
```

Comment:
avoid doing this, instead you should be doing this 

upper in the code replace

```
if data == nil {
                    
                    //no body, but a valid response
                    completion(response: httpResponse, body: nil, error: nil)
                    return
                }
```

with this:

```
guard let data = data else {
                    
                    //no body, but a valid response
                    completion(response: httpResponse, body: nil, error: nil)
                    return
                }
```

so after data data won't be an optional anymore so this `let (json, error) = JSON.parse(data!)` becomes `let (json, error) = JSON.parse(data)`